deploy/operator.yaml

```diff
@@ -15,11 +15,8 @@ spec:
       serviceAccountName: tekton-operator
```

Comment:
Should restructure the `/deploy` or `/config`,  add number as prefix of files, like 500-operator.yaml, it will keep the order when run `ko apply`,  avoid exception caused by the wrong order.

Reply:
So far, the CRD installation still goes before the operator installation as prerequisite. We may need to do the ordering later if we decide to move the CRD into the operator installation as well.